src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/DashboardTopBanner/DashboardTopBanner.test.tsx

```diff
@@ -0,0 +1,213 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Curious why there's a new test file for this, since the other dashboard top banner test variations were in `Dashboard.test.tsx`?